src/cute_shader/cute_shaderc.cpp

```diff
@@ -106,8 +106,14 @@ static bool write_bytecode_struct(
 			fprintf(file, " \"%s\",", shader_info->image_names[i]);
 		}
 		fprintf(file, "\n};\n");
+		fprintf(file, "static int %s%s_image_binding_slots[%d] = {\n   ", var_name, suffix, shader_info->num_images);
```

Comment:
`const int` for consistency with the rest.

Also some **C** compilers (e.g: Clang) are strict about what is considered "constant expression" for static initializer used in `CF_ShaderBytecode` later